databend_py/uploader.py

```diff
@@ -4,14 +4,16 @@
 import uuid
 import json
 import time
+import gzip
 
 
 class DataUploader:
-    def __init__(self, client, settings, default_stage_dir='@~', debug=False):
+    def __init__(self, client, settings, default_stage_dir='@~', debug=False, compress=False):
         # TODO: make it depends on Connection instead of Client
         self.client = client
         self.settings = settings
         self.default_stage_dir = default_stage_dir
+        self._compress = compress
```

Comment:
Need to update the connection.md for `compress` parameter , may be next pr.

Reply:
thank you for reminding, currently this parameter might not be that stable, I'll add this paramter in the next pr after add more tests about it